DashWallet/Sources/Models/DWBalanceNotifier.m

```diff
@@ -92,28 +93,43 @@ - (void)walletBalanceDidChangeNotification:(NSNotification *)note {
 
     if (self.balance < wallet.balance) {
         const BOOL notificationsEnabled = [DWGlobalOptions sharedInstance].localNotificationsEnabled;
-        NSString *noteText = [NSString stringWithFormat:
-                                           NSLocalizedString(@"Received %@ (%@)", nil),
-                                           [priceManager stringForDashAmount:wallet.balance - self.balance],
-                                           [priceManager localCurrencyStringForDashAmount:wallet.balance - self.balance]];
+        UInt64 received = wallet.balance - self.balance;
+        NSString *noteText;
+        NSString *identifier;
+        UNNotificationSound *sound;
+        Boolean isCrowdNode = received == ApiCodeDepositReceived + CrowdNodeObjcWrapper.apiOffset;
+        
+        if (isCrowdNode) {
```

Comment:
For CrowdNode, we want `Deposit received` notification instead of the coinflip notification, and to show it if the app is in the foreground as well.